core/server/src/test/java/alluxio/worker/block/BlockWorkerTest.java

```diff
@@ -294,6 +294,20 @@ public void getVolatileBlockMetaTest() throws Exception {
   }
 
   /**
+   * Tests the {@link BlockWorker#getBlockMeta(long)} method.
+   *
+   * @throws Exception if the getVolatileBlockMeta check fails
+   */
+  @Test
+  public void getBlockMetaTest() throws Exception {
+    long sessionId = mRandom.nextLong();
```

Comment:
my personal opinion: in general i am concerned to have randomness in unit tests unless it is designed to be a randomized test. The reason is we want the unit tests to be in general repeatable.